1-callbacks/solution.js

```diff
@@ -31,17 +31,54 @@ node solution.js name1 name2 name3
 5. another challenge is: after you solve the challenge using callback style, in another file promisify the callback and solve it again
 ** give a look to node.js util.promisify, avoid to alter the validate-user.file **
 */
+const validateUser = require("./validate-user.js");
 
 function solution() {
-    // YOUR SOLUTION GOES HERE
-
     // you get your 5 names here
+    const names = ["Carlos", "Andrea", "Miguel", "Juan", "Richard"];
+
+    // extra challange #1 read from terminal arguments
+    if(process.argv.length > 2){
+        // just add them, so that I don't have to type a bunch in terminal. 
+        names.push(...process.argv.slice(2));
+    }
+    
+    const okNames = [];
+    const errNames = [];
```

Comment:
You can improve readability by using names like.

`authorizedUsers`
`unAuthorizedUsers`